migrations/0008_affiliates_table.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE affiliates
+(
+    account_name VARCHAR(13) PRIMARY KEY UNIQUE,
+    affiliate_id VARCHAR NOT NULL
```

Comment:
maybe better to store `affiliate_id` in the `users` table?

Reply:
algys said it's better to create different table for affiliates.